src/cmd/invites/invites.go

```diff
@@ -13,6 +13,7 @@ import (
 var InvitesCmd = &cobra.Command{
 	Use:     "invites",
 	GroupID: groups.AccountsGroup.ID,
+	Aliases: []string{"invite"},
```

Comment:
Nice! Do we have singular form in all other commands? If not please add 😁

Reply:
yes, that's the only one I missed in the previous PR.